include/deal.II/base/partitioner.templates.h

```diff
@@ -329,6 +329,43 @@ namespace Utilities
         return a;
       }
 
+      // In the import_from_ghosted_array_finish we need to calculate maximal
+      // and minimal value on number types, which is not straight forward for
+      // complex numbers. Therfore, comparison of complex numbers is
+      // prohibited and throw an assert.
+      template <typename Number>
+      Number
+      get_min(const Number a, const Number b)
+      {
+        return std::min(a, b);
+      }
+
+      template <typename Number>
+      std::complex<Number>
+      get_min(const std::complex<Number> a, const std::complex<Number>)
+      {
+        AssertThrow(false,
+                    ExcMessage("VectorOperation::min max not"
```

Comment:
gives build error

Reply:
I think that what @masterleinad was referring to is that you should write this assertion as
```
AssertThrow(false, 
       ExcMessage("VectorOperation::min not "
       "implemented for complex numbers"));
```
rather than return `std::max(a)`.

Note as well the extra space at the end of the first line of the assertion message. If this is not here then `VectorOperation::min notimplemented for complex numbers` will be printed.